README.md

```diff
@@ -42,14 +42,21 @@ On first run, the robot will execute an example program. On first run, this will
 
 In order to run competition matches you'll need to:
 
-1. Create a file `robot_mode.txt` in the root of the repo which contains just
+1. Install the third party libraries the teams are depending on, into the same
```

Comment:
I think it might be useful to mention here that this can be configured a la https://studentrobotics.org/docs/competition-simulator/#changing-your-version-of-python

Reply:
7563adc9923a2fd028b6e7c44e6b7894d77c180a